### DISCUSSION TITLE
Default for LinkToAnotherRecord [May 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5760] [Discord Threads]
> 🙋‍♂️ **Nils**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### <@253601577271689216> Not possible. What is your use case?  
#  
> 🙋‍♂️ **Nils**  
#### I have a table with email addresses and another table with emails (message, subject, sent at, sender, receiver, etc.). For the sender and receiver column is a link to the email address table. I have now 110.000 email I need to import to NocoDB via the API.   
####    
#### My current approach is to first add the the emails with a POST request and after adding them running every email and every participant (sender and receiver) a PATCH request to link the email address a email row. However, these are ~300k - 400k HTTP requests and it will take multiple days to import the data.   
####    
#### Especially because there no bulk option for linking records it takes a long time. Having the option to directly pass the ID of the linked record would allow me to use the bulk option when adding new rows because for adding rows there is a bulk option.  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### <@253601577271689216> Thanks for the details. We are considering simplifying mechanisms to work with data using APIs; & bulk link updates should be possible in next API related change iteration. Can't commit on timelines, but can assure, this is on our roadmap  
#  
> 🙋‍♂️ **Nils**  
#### Cool, thanks <3  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1098606137462501386)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
